### ISSUE TITLE
[Crunchyroll] Error: Unable to extract title

### ISSUE THREAD

**Issue by lindraden** [ytdl-org/youtube-dl#21500]
## Checklist

- [ ] I'm reporting a broken site support issue
- [x] I've verified that I'm running youtube-dl version **2019.06.21**
- [x] I've checked that all provided URLs are alive and playable in a browser
- [x] I've checked that all URLs and arguments with special characters are properly quoted or escaped
- [x] I've searched the bugtracker for similar bug reports including closed ones
- [x] I've read bugs section in FAQ

## Verbose log

```
E:\Crunchy>youtube-dl.exe https://www.crunchyroll.com/yu-gi-oh-gx --verbose
[debug] System config: []
[debug] User config: []
[debug] Custom config: []
[debug] Command-line args: ['https://www.crunchyroll.com/yu-gi-oh-gx', '--verbose']
[debug] Encodings: locale cp1252, fs mbcs, out cp850, pref cp1252
[debug] youtube-dl version 2019.06.21
[debug] Python version 3.4.4 (CPython) - Windows-10-10.0.17134
[debug] exe versions: ffmpeg N-93726-g7eba264513
[debug] Proxy map: {}
[crunchyroll:playlist] yu-gi-oh-gx: Downloading webpage
ERROR: Unable to extract title; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type  youtube-dl -U  to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
Traceback (most recent call last):
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmppgbwk_g4\build\youtube_dl\YoutubeDL.py", line 796, in extract_info
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmppgbwk_g4\build\youtube_dl\extractor\common.py", line 530, in extract
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmppgbwk_g4\build\youtube_dl\extractor\crunchyroll.py", line 666, in _real_extract
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmppgbwk_g4\build\youtube_dl\extractor\common.py", line 1014, in _html_search_regex
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmppgbwk_g4\build\youtube_dl\extractor\common.py", line 1005, in _search_regex
youtube_dl.utils.RegexNotFoundError: Unable to extract title; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type  youtube-dl -U  to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
```

## Description

Trying to get this working with Crunchyroll **Playlists**. Not sure what I'm missing though. I've tried plugging in my username and password in it too. It does work if I put in the episode in the link (eg https://www.crunchyroll.com/yu-gi-oh-gx/episode-1-yugis-heir-684069) but the playlist itself does not and gives me Error matching title. 
I've tried this on both my Windows and my Mac. Both updated using youtube-dl -U.

**Comment by lindraden** [comment]
How is this issue a duplicate of https://github.com/ytdl-org/youtube-dl/pull/15840??

An explanation would be nice.

**Comment by remitamine** [comment]
> I've searched the bugtracker for similar bug reports including **closed ones**

**Comment by SerenadeX** [comment]
I am seeing this problem too.  It looks like this may have been fixed as discussed in #21291, but I am having this issue and there are two more duplicate issue of people having the same problem.  Can we confirm that this was fixed please?
